openapi/paylocity/types.bal

```diff
@@ -1002,31 +1002,31 @@ public type EmployeeHomeaddress record {
 # The staged employee model
 public type StagedEmployee record {
     # Add up to 19 direct deposit accounts in addition to the main direct deposit account. IMPORTANT: A direct deposit update will remove ALL existing main and additional direct deposit information in WebPay and replace with information provided on the request. GET API will not return direct deposit data.
-    StagedemployeeAdditionaldirectdeposit[] additionalDirectDeposit?;
+    StagedemployeeAdditionaldirectdeposit[4] additionalDirectDeposit?;
     # Add setup values used for employee benefits integration, insurance plan settings, and ACA reporting.
-    StagedemployeeBenefitsetup[] benefitSetup?;
+    StagedemployeeBenefitsetup[1] benefitSetup?;
     # Employee birthdate. Common formats include *MM-DD-CCYY*, *CCYY-MM-DD*.
     string? birthDate?;
     # Up to 8 custom fields of boolean (checkbox) type value.
-    EmployeeCustombooleanfields[] customBooleanFields?;
+    EmployeeCustombooleanfields[8] customBooleanFields?;
     # Up to 8 custom fields of the date type value.
-    EmployeeCustomdatefields[] customDateFields?;
+    EmployeeCustomdatefields[8] customDateFields?;
     # Up to 8 custom fields of the dropdown type value.
-    EmployeeCustomdropdownfields[] customDropDownFields?;
+    EmployeeCustomdropdownfields[8] customDropDownFields?;
     # Up to 8 custom fields of numeric type value.
-    EmployeeCustomnumberfields[] customNumberFields?;
+    EmployeeCustomnumberfields[8] customNumberFields?;
```

Comment:
Hope this is not going to be a breaking change since this probably do not have any user

Reply:
I guess no issue in here.